src/TestBootstrap.lua

```diff
@@ -108,6 +116,10 @@ function TestBootstrap:run(roots, reporter, otherOptions)
 		end
 	end
 
+	table.sort(modules, function(a, b)
```

Comment:
Why move this into run? It's entirely possible to use TestEZ without calling run (as it's just meant to be a shortcut). Also, I think the plan is to get rid of TestBootstrap.run and use TestEZ.run instead though that's going to be a little while.

Reply:
Because getModule is called multiple times, one time per test root, but sort order need to be global for entire test tree. I moved it to TestPlanner as it makes more sense to be there

Reply:
Ah, good point.